src/evt.js

```diff
@@ -36,11 +28,29 @@ evt.prototype.raise = function() {
 };
 
 function evtInit(element) {
-
-  // handle the element/selector
-
-  return new evt();
+  if (!element || (element instanceof Element) === false) {
+    throw new Error('You must supply a DOM element or selector to evt');
+  }
+
```

Comment:
If element is a selector, it's going to need to be a string. If not, it should be an Element, or a collection of Elements.